### PR TITLE
{bio}[GCC/11.3.0] PLINK v2.00a3.6

### PR DESCRIPTION
**Pull request by benczaja**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/3adca63303de6f05033ce4e33069ed90 for a full test report.

File: easybuild/easyconfigs/p/PLINK/PLINK-2.00a3.6-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+]
+
+start_dir = '2.0'
+
+# ignore suggested Makefile in build_dynamic, the dev Makefile is better
+buildopts = 'BLASFLAGS="-lflexiblas" BLASFLAGS64="-lflexiblas"'
+buildopts += ' LDFLAGS="$LIBS -ldeflate -lz -lzstd" ZLIB="" CSRC="include/SFMT.c"'
+buildopts += ' BASEFLAGS="$OPTFLAGS -DZSTD_MULTITHREAD"'
+
+files_to_copy = ['bin', (['include/*.h'], 'include'), 'ReadMe.md']
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PLINK/PLINK-2.00a3.6-GCC-11.3.0.eb:30]: @benczaja any reason why you're no longer including `LICENSE`?

Comment (benczaja) [reply easybuild/easyconfigs/p/PLINK/PLINK-2.00a3.6-GCC-11.3.0.eb]: @verdurin It looks like the LICENSE (GNU General Public License v3) has been renamed to COPYING in the new version. I have just added it now.

Comment (verdurin) [reply easybuild/easyconfigs/p/PLINK/PLINK-2.00a3.6-GCC-11.3.0.eb]: I think there are separate files for the `1.9` and `2.0` trees.
Anyway, thanks.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/2acb142e10154fd0c9febd0a17a0155c for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @benczaja!

**Pull request merged**